topics/go/algorithms/sorting/selection/selection.go

```diff
@@ -0,0 +1,24 @@
+// Package selectionsort implement of Selection Sort algorithm in Go.
+package selectionsort
+
+// selectionSortIterative takes a random list of numbers and uses the
+// `iterative` process to sort it and return the sorted list.
+func selectionSortIterative(randomList []int) []int {
+
+	// Loop until the random list will be sorted.
+	for i := range randomList {
+		index := i
+
+		// Continue the loop until to found the smallest number in the list.
```

Comment:
Read that comment out loud.

Reply:
Explain the what and why

Reply:
it's not the smallest number in the list, smallest compared to randomList[i]. How can we improve the variable names for i and j?

Reply:
Would you think this sentence could be fine and clear? 

```
// Loop to find the smallest number while compared to randomList[i], 
// If the smallest number is found, the index value changed to that index.
```
and to improve the variable names, instead, to have `i` and `j` we could write something like `leftIdx` and `smallestIdx`